.github/workflows/start_benchmarks.yml

```diff
@@ -38,7 +46,10 @@ jobs:
     if: ${{ (github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs') || github.event_name == 'workflow_dispatch' }}
     strategy:
       matrix:
-        command: [boolean_bench, shortint_bench, integer_bench, integer_multi_bit_bench, pbs_bench, wasm_client_bench]
+        command: [ boolean_bench, shortint_bench,
+                   integer_bench, integer_multi_bit_bench,
+                   signed_integer_bench, signed_integer_multi_bit_bench,
+                   pbs_bench, wasm_client_bench ]
```

Comment:
file changes need to take into account the new signed benchmarks below ?

Reply:
You're right !